src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.tpl.html

```diff
@@ -19,7 +19,7 @@
           ng-change="taskDefinitionIdChanged()"
           autofocus="{{autofocus.onTaskDefinitionSelect}}"
           class="form-control input-md">
-          <option value="" ng-hide="!isTaskDefMode">All task definitions</option>
+          <option disabled value="" ng-hide="!isTaskDefMode">Task Definitions</option>
```

Comment:
Dont want to use disabled here - people need to be able to select this.
Change to ng-if rather than ng-hide - that is what is failing here